integration_tests/openvino/basic_full_flow.py

```diff
@@ -0,0 +1,24 @@
+import numpy as np
```

Comment:
There should be much more extensive integration testing -- covering each of the supported use cases (e.g. CNNs, Transformers for LLMs, RNNs, etc.)

Reply:
Now we are able to check each operation support using existing tests. So no need to implement separate set of tests for openvino.
I am managed to adopt the existing tests despite there is no eager execution for openvino. For this, I run inference for `ov::Output` resulting node in `convert_tensor_to_numpy` and it provides the requested output data computed using openvino.